src/app/features/vendor/vendor-routing.module.ts

```diff
@@ -6,6 +6,7 @@ import { VendorCreationComponent } from './vendor-creation/vendor-creation.compo
 const routes: Routes = [
   { path: '', component: VendorListingComponent },
   { path: 'creation', component: VendorCreationComponent },
+  { path: 'edit', component: VendorCreationComponent },
```

Comment:
Add vendor id in path to specify the selected vendor.
eg: edit/id

Reply:
done